packages/nuxt/src/app/plugins/view-transitions.client.ts

```diff
@@ -35,6 +35,7 @@ export default defineNuxtPlugin((nuxtApp) => {
       changeRoute()
       return promise
     })
+    nuxtApp.callHook('page:view-transition:start', transition)
```

Comment:
```suggestion
    await nuxtApp.callHook('page:view-transition:start', transition)
```
We probably need to await the hook